test/src/test/kotlin/graphql/nadel/tests/next/fixtures/hydration/virtual/HydrationMakesVirtualFieldTestSnapshot.kt

```diff
@@ -0,0 +1,136 @@
+                |     "graphStore_query": {
+                |       "edges": [
+                |         {
+                |           "batch_hydration__node__nodeId": "ari:cloud:jira::issue/1",
+                |           "__typename__batch_hydration__node": "GraphStoreQueryEdge"
+                |         }
+                |       ]
+                |     }
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResults = listOfJsonStrings(
+                ),
+            ),
+            ExpectedServiceCall(
+                service = "jira",
+                query = """
+                | {
+                |   issuesByIds(ids: ["ari:cloud:jira::issue/1"]) {
+                |     key
+                |     batch_hydration__node__id: id
+                |   }
+                | }
+                """.trimMargin(),
+                variables = " {}",
+                result = """
+                | {
+                |   "data": {
+                |     "issuesByIds": [
+                |       {
+                |         "key": "GQLGW-1",
+                |         "batch_hydration__node__id": "ari:cloud:jira::issue/1"
+                |       }
+                |     ]
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResults = listOfJsonStrings(
+                ),
+            ),
+            ExpectedServiceCall(
+                service = "work",
+                query = """
+                | {
+                |   __typename__hydration__businessReport_findRecentWorkByTeam: __typename
+                | }
+                """.trimMargin(),
+                variables = " {}",
+                result = """
+                | {
+                |   "data": {
+                |     "__typename__hydration__businessReport_findRecentWorkByTeam": "Query"
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResults = listOfJsonStrings(
+                ),
+            ),
+        )
+
+    /**
+     * ```json
+     * {
+     *   "data": {
+     *     "businessReport_findRecentWorkByTeam": {
+     *       "edges": [
+     *         {
+     *           "node": {
+     *             "key": "GQLGW-1"
+     *           }
+     *         }
+     *       ]
+     *     }
+     *   }
+     * }
+     * ```
+     */
+    override val result: ExpectedNadelResult = ExpectedNadelResult(
+            result = """
+            | {
+            |   "data": {
+            |     "businessReport_findRecentWorkByTeam": {
+            |       "edges": [
+            |         {
+            |           "node": {
+            |             "key": "GQLGW-1"
+            |           }
+            |         }
+            |       ]
+            |     }
+            |   }
+            | }
+            """.trimMargin(),
+            delayedResults = listOfJsonStrings(
+            ),
+        )
+}
```

Comment:
Shall we add tests for other type shapes - like a list of objects, or simply an object?
This would showcase this possibility and make sure the code is flexible to support those.

Reply:
Yep, I need to add some more tests. Thanks for the review!